examples/examples_dotnet_test.go

```diff
@@ -90,21 +90,21 @@ func TestDotNetOss012(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
-func TestDotNetRemoteBackend(t *testing.T) {
-	test := getDotNetBaseOptions(t).
-		With(integration.ProgramTestOptions{
-			Dir: path.Join(getCwd(t), "remote-backend-dotnet"),
-			Config: map[string]string{
-				"organization":  getRemoteBackendOrganization(t),
-				"workspaceName": "dev",
-			},
-			Secrets: map[string]string{
-				"tfeToken": getRemoteBackendToken(t),
-			},
-		})
-
-	integration.ProgramTest(t, &test)
-}
+// func TestDotNetRemoteBackend(t *testing.T) {
```

Comment:
There's `t.Skip("TODO[pulum/pulumi-terraform#12345]")` form that's better as it's more explicit that there's a test that's currently not working.